pandas/tests/series/test_ufunc.py

```diff
@@ -439,3 +439,13 @@ def test_outer():
 
     with pytest.raises(NotImplementedError, match=tm.EMPTY_STRING_PATTERN):
         np.subtract.outer(s, o)
+
+
+def test_npmul():
+    # GH26650
+    data = [-1, 1, 10]
+    df = pd.DataFrame({"nums": data})
+    dfT = df.T
```

Comment:
Please avoid these functions

Reply:
Thanks for reviewing and the feedback. I removed these lines and declared everything directly on the assert statement. Just for my understanding, these lines needed to be removed since they introduce too many variables?

Reply:
No, we try to define tests in a way that they do not rely on functions that are not tested itself. E.g. .T was not necessary to reproduce the behavior.

You can do it like:

df=...
result=...
expected=...
arm.assert_frame_equal(result, expected)

Reply:
Oh ok, I see. Thanks again! I modified the PR to reflect this.